.github/workflows/live-deploy.yml

```diff
@@ -0,0 +1,30 @@
+# An Action to build Web Arcs and deploy to https://live.arcs.dev
+name: live-deploy
+
+on:
+  push:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Run Typescript Tests
+        run: |
+          tools/sigh webpack
+          tools/sigh test --all
+
+      - name: Run deployment script
```

Comment:
How do you make sure that this step waits for the previous step?

Reply:
Generally, jobs run in parallel, but steps run sequentially [1]. The way this is configured, the deploy script will wait for the build step. 

[1] https://docs.github.com/en/free-pro-team@latest/actions/learn-github-actions/introduction-to-github-actions#steps